Lib/textwrap.py

```diff
@@ -365,7 +380,7 @@ def fill(self, text):
 
 # -- Convenience interface ---------------------------------------------
 
-def wrap(text, width=70, **kwargs):
+def wrap(text, width=70, cjk=False, **kwargs):
```

Comment:
There is not need to repeat cjk here, there is already a generic **kwargs. Same for other functions below.